regression: mismatched types (panic info hook stuff)

- https://crater-reports.s3.amazonaws.com/beta-1.81-2/beta-2024-07-26/reg/yacurses-0.2.4/log.txt
- https://crater-reports.s3.amazonaws.com/beta-1.81-2/beta-2024-07-26/gh/SNIAPA.screeps-ballerbot/log.txt
- https://crater-reports.s3.amazonaws.com/beta-1.81-2/beta-2024-07-26/reg/crashy-0.3.0/log.txt
- https://crater-reports.s3.amazonaws.com/beta-1.81-2/beta-2024-07-26/reg/static_init-1.0.3/log.txt
- https://crater-reports.s3.amazonaws.com/beta-1.81-2/beta-2024-07-26/reg/tezos-smart-rollup-entrypoint-0.2.2/log.txt
- https://crater-reports.s3.amazonaws.com/beta-1.81-2/beta-2024-07-26/gh/WalrusGumboot.linen/log.txt
- https://crater-reports.s3.amazonaws.com/beta-1.81-2/beta-2024-07-26/gh/pokir.learn-rust-snake/log.txt
- https://crater-reports.s3.amazonaws.com/beta-1.81-2/beta-2024-07-26/reg/tezos-smart-rollup-0.2.2/log.txt
- https://crater-reports.s3.amazonaws.com/beta-1.81-2/beta-2024-07-26/reg/tmuxstatus-1.0.1/log.txt

```
[INFO] [stdout] error[E0308]: mismatched types
[INFO] [stdout]    --> src/lib.rs:122:36
[INFO] [stdout]     |
[INFO] [stdout] 122 |       std::panic::set_hook(replace(&mut self.old_hook, Box::new(|_| {})));
[INFO] [stdout]     |                            ------- ^^^^^^^^^^^^^^^^^^ expected `PanicHookInfo<'_>`, found `PanicInfo<'_>`
[INFO] [stdout]     |                            |
[INFO] [stdout]     |                            arguments to this function are incorrect
[INFO] [stdout]     |
[INFO] [stdout]     = note: expected mutable reference `&mut Box<dyn for<'a, 'b> std::ops::Fn(&'a PanicHookInfo<'b>) + Send + Sync>`
[INFO] [stdout]                found mutable reference `&mut Box<(dyn for<'a, 'b> std::ops::Fn(&'a PanicInfo<'b>) + Send + Sync + 'static)>`
```

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -54,9 +54,9 @@ use core::{
   mem::replace,
   num::NonZeroU8,
   ops::*,
-  panic::PanicInfo,
   sync::atomic::{AtomicBool, Ordering},
 };
+use std::panic::PanicInfo;
 
 mod curses_common;
 use curses_common::*;

